js/render/draw_symbol.js

```diff
@@ -113,24 +113,26 @@ function drawSymbol(painter, layer, posMatrix, tile, bucket, bufferGroups, isTex
     var defaultSize = isText ? 24 : 1;
     var fontScale = size / defaultSize;
 
-    var extrudeScale, s, gammaScale;
-    if (alignedWithMap) {
-        s = pixelsToTileUnits(tile, 1, painter.transform.zoom) * fontScale;
-        gammaScale = 1 / Math.cos(tr._pitch);
-        extrudeScale = [s, s];
-    } else {
-        s = painter.transform.altitude * fontScale;
-        gammaScale = 1;
-        extrudeScale = [ tr.pixelsToGLUnits[0] * s, tr.pixelsToGLUnits[1] * s];
-    }
+    var s1 = painter.transform.altitude * fontScale;
+    var s2 = pixelsToTileUnits(tile, 1, painter.transform.zoom) * fontScale;
+    var gammaScale = 1; // 1 / Math.cos(tr._pitch);
```

Comment:
Yeah, I need to vet this further -- it adjusts the SDF sharpness when pitching the map but the previous calculation was not giving good results.